examples/lband_decode/lband_decode.cc

```diff
@@ -9,11 +9,15 @@
 
 #include <point_one/fusion_engine/messages/gnss_corrections.h>
 #include <point_one/fusion_engine/parsers/fusion_engine_framer.h>
+#include <point_one/rtcm/rtcm_framer.h>
 
 using namespace point_one::fusion_engine::messages;
 using namespace point_one::fusion_engine::parsers;
+using namespace point_one::rtcm;
 
-static std::ofstream out_stream;
+static constexpr size_t READ_SIZE_BYTES = 1024;
+static constexpr size_t FRAMER_BUFFER_BYTES = 10240;
```

Comment:
That's a really big RTCM framer buffer. This is also using the same constant for the RTCM and FE framers, which is a bit confusing. The max RTCM message size is 1023 bytes. The max FE L-band message is (FE header + message) + 504 B (max PMP user data size).